dockerfiles/Dockerfile

```diff
@@ -0,0 +1,23 @@
+from ros:noetic
+
+run apt update && apt install -qy curl
+
+env DEBIAN_FRONTEND=noninteractive
+
+run curl http://robotpkg.openrobots.org/packages/debian/robotpkg.key | apt-key add - \
+  && curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | sudo apt-key add -
+
+run echo "deb [arch=amd64] http://robotpkg.openrobots.org/packages/debian/pub focal robotpkg" > /etc/apt/sources.list.d/robotpkg.list
+run apt update && apt install -qy \
+  robotpkg-py38-hpp-manipulation-corba robotpkg-py38-qt5-hpp-gepetto-viewer \
+  robotpkg-py38-qt5-hpp-gui robotpkg-py38-qt5-hpp-plot \
+  ros-noetic-pr2-description robotpkg-py38-hpp-environments \
+  oxygen-icon-theme \
+  && rm -rf /var/lib/apt/lists/*
+
+copy hpp_config.sh hpp_entrypoint.sh /
+run chmod a+x /hpp_entrypoint.sh
+run echo "[pyplugins]\nhpp.gui=true" >> /opt/openrobots/etc/gepetto-gui/settings.conf
+run echo "[pyplugins]\nhpp.gui=true" >> /opt/openrobots/etc/gepetto-gui/basic.conf
+
+entrypoint [ "/hpp_entrypoint.sh" ]
```

Comment:
```suggestion
FROM ros:noetic

RUN --mount=type=cache,sharing=locked,target=/var/cache/apt \
    --mount=type=cache,sharing=locked,target=/var/lib/apt \
    apt-get update -y && DEBIAN_FRONTEND=noninteractive apt-get install -qqy \
    curl \
 && curl http://robotpkg.openrobots.org/packages/debian/robotpkg.key | apt-key add - \
 && echo "deb [arch=amd64] http://robotpkg.openrobots.org/packages/debian/pub focal robotpkg" > /etc/apt/sources.list.d/robotpkg.list \
 && apt-get update -y && DEBIAN_FRONTEND=noninteractive apt-get install -qqy \
    oxygen-icon-theme \
    robotpkg-py38-hpp-environments \
    robotpkg-py38-hpp-manipulation-corba \
    robotpkg-py38-qt5-hpp-gepetto-viewer \
    robotpkg-py38-qt5-hpp-gui \
    robotpkg-py38-qt5-hpp-plot \
    ros-noetic-pr2-description

RUN echo "[pyplugins]\nhpp.gui=true" >> /opt/openrobots/etc/gepetto-gui/settings.conf \
 && echo "[pyplugins]\nhpp.gui=true" >> /opt/openrobots/etc/gepetto-gui/basic.conf

ENV PATH=/opt/openrobots/bin:${PATH} \
    LD_LIBRARY_PATH=/opt/openrobots/lib:${LD_LIBRARY_PATH} \
    PYTHONPATH=/opt/openrobots/lib/python3.8/site-packages:${PYTHONPATH} \
    ROS_PACKAGE_PATH=/opt/openrobots/share:${ROS_PACKAGE_PATH} \
    CMAKE_PREFIX_PATH=/opt/openrobots:${CMAKE_PREFIX_PATH} \
    PKG_CONFIG_PATH=/opt/openrobots:${PKG_CONFIG_PATH}
```

This would make the image smaller and faster to build. But this require buildkit (so `export DOCKER_BUILDKIT=1`).
Also this remove the need for the config and entrypoint files